zerver/models.py

```diff
@@ -3900,6 +3900,7 @@ class AbstractRealmAuditLog(models.Model):
     # Values are chosen to be 10000 greater than the value in RealmAuditLog.
     REMOTE_SERVER_CREATED = 10215
     REMOTE_SERVER_PLAN_TYPE_CHANGED = 10204
+    REMOTE_SERVER_DEACTIVATED_CHANGED = 10205
```

Comment:
This should be 10201, right?  Did you miss the comment above, or is it not clear enough about how values should be selected?  (In general, one should always read the comments above a block before modifying it, especially for something in the data model like this).

Also, I think the right name is `REMOTE_SERVER_DEACTIVATED` given the conventions here.

Reply:
I read that comment but I didn't automatically translate it to "the value should be exactly 10000 more than the corresponding value for the same purpose used in `RealmAuditLog`". I'll modify the comment to be more explicit here. Thanks for pointing this out!